pkgs/applications/misc/plasma-applet-commandoutput/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, cmake, extra-cmake-modules, plasma-framework, kwindowsystem, plasma-pa, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "plasma-applet-commandoutput";
+  version = "2022-01-11";
+
+  src = fetchFromGitHub {
+    owner = "Zren";
+    repo = "plasma-applet-commandoutput";
+    rev = "d4c93a1fad82ff4d2fcc622f592b28a60878f49f";
+    sha256 = "0hg5i8yq159263yz7jyhpi893c5c2kl59w03p9nnpw656xr6102b";
+  };
+
+  # Adds the CMakeLists.txt not provided by upstream
+  patches = [ ./cmake.patch ];
+  postPatch = "rm build";
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+  buildInputs = [ plasma-framework kwindowsystem plasma-pa ];
+
+  dontWrapQtApps = true;
+
+  meta = with lib; {
```

Comment:
You're missing the license field.

Reply:
There is no license in the repository yet. I'll open an issue about that!